packages/components/core/styles/common/_select.scss

```
@@ -131,6 +131,11 @@
         }
     }
 
+    & kbq-select-trigger,
+    & [kbq-select-trigger] {
```

& здесь необязательно проставлять, только если ты явно хочешь показать принадлежность. Но SCSS сам проставит иерархию

```suggestion
    [kbq-select-trigger] {
```

---

да я чет привык & ставить...